cooperator/models/subscription_request.py

```diff
@@ -896,3 +908,19 @@ def put_on_waiting_list(self):
             )
         self._send_waiting_list_mail()
         self.write({"state": "waiting"})
+
+    @api.constrains("ordered_parts")
+    def _check_ordered_parts_quantity(self):
+        for sub_req in self:
+            if (
+                sub_req.ordered_parts
+                < sub_req.share_product_id.product_tmpl_id.minimum_quantity
+            ):
+                raise ValidationError(
+                    _(
+                        "The number of shares must be more than the defined minimum "
+                        f"quantity of this share. For {sub_req.share_product_id.short_name},"
+                        " the minimum is"
+                        f"{sub_req.share_product_id.product_tmpl_id.minimum_quantity}."
+                    )
```

Comment:
f-strings prevent localization (as they would change the source string). the substitution must be done after the translation.
```suggestion
                    _(
                        "The number of shares must be more than the defined minimum "
                        "quantity of this share. For {share_type}, the minimum is "
                        "{minimum_quantity}."
                    ).format(
                        share_type=sub_req.share_product_id.short_name,
                        minimum_quantity=sub_req.share_product_id.product_tmpl_id.minimum_quantity
                    )
```